test/unit/kubernetes-deploy/statsd_test.rb

```diff
@@ -18,5 +18,6 @@ def test_build_when_statsd_addr_env_present_but_statsd_implementation_is_not
     ENV['STATSD_ADDR'] = original_addr
     ENV['STATSD_IMPLEMENTATION'] = original_impl
     ENV['STATSD_DEV'] = original_dev
+    KubernetesDeploy::StatsD.build
```

Comment:
This sets global state on `::StatsD`, so if we don't redo it after resetting the env vars, the other tests will try to use the UDP backend and throw errors (because this test doesn't actually fully configure it).